virtcontainers/kata_proxy.go

```diff
@@ -24,6 +24,7 @@ func (p *kataProxy) consoleWatched() bool {
 
 // start is kataProxy start implementation for proxy interface.
 func (p *kataProxy) start(sandbox *Sandbox, params proxyParams) (int, string, error) {
+	sandbox.Logger().Info("Starting no built-in Kata Proxy")
```

Comment:
I don't understand this message as this function does start a `kata-proxy`.

Reply:
@sboeuf  ^^

Reply:
@jodh-intel maybe the formulation is not proper, but the idea is to say that we are starting the regular Kata proxy. Because we also have a built-in proxy that we can start, but in this case, this is a simple go routine. And this also compares with the no proxy which means that we're actually not starting any proxy.

Reply:
Ah - in which case, I'd reword to something like the following to make that clearer:

> Starting regular Kata proxy rather than built-in

Reply:
+1, /cc @devimc